src/sciwrappers.h

```diff
@@ -56,162 +57,162 @@ void 				sci_set_current_position	(ScintillaObject *sci, gint position, gboolean
 
 gint				sci_get_selection_start		(ScintillaObject *sci);
 gint				sci_get_selection_end		(ScintillaObject *sci);
-void 				sci_replace_sel				(ScintillaObject *sci, const gchar *text);
+void 				sci_replace_sel			(ScintillaObject *sci, const gchar *text);
```

Comment:
Please don't rewrite unrelated indentation, even if it was incorrect (and here it's not, Geany uses a tab width of 4, which aligns well).  Changes like this make it a lot harder to read the actual, meaningful changes, and should be made in a specific commit if at all.

Reply:
To be fair, aligning stuff like this (GNU/GTK+-style) is a terrible idea, especially when tabs are used. Not sure why this one file in Geany uses this alignment style.

Reply:
Yeah, but to return it to its original state only needs `git checkout src/sciwrappers.h`.  There was so much noise I didn't even notice that there was an actual change. @b4n did better.

Reply:
The indentation is horrid in this file - I tried to be nice, it won't happen again.

Reply:
> The indentation is horrid in this file

:+1: 

At some point I'll probably make a PR to reformat this header to follow Geany's coding style.